modules/admin_manual/pages/enterprise/file_management/files_lifecycle.adoc

```diff
@@ -0,0 +1,126 @@
+= File Lifecycle Management
+:toc: right
+
```

Comment:
Please add a section (as we have in other longer pages):

== Introduction

Reply:
@pmaier1